bootcamp/users/views.py

```diff
@@ -7,14 +7,21 @@
 from django.views.generic import DetailView, ListView, RedirectView, UpdateView
 from django.conf import settings as django_settings
 from .models import User
+from ..news.models import News
```

Comment:
This import is a bad pattern, it is preferable to use base project imports using absolute references.